ClickHouse.Client/Types/ClickHouseType.cs

```diff
@@ -8,6 +8,12 @@ internal abstract class ClickHouseType
 
         public abstract Type FrameworkType { get; }
 
+        public abstract string ToHttpParameter(object value);
```

Comment:
This should really be in separate class (like `BinaryStreamWriter`, but for http)

Reply:
done